src/commands/commerce/extension/map.ts

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { flags, SfdxCommand } from '@salesforce/command';
+import { Messages, Org, SfdxError } from '@salesforce/core';
+import { forceDataSoql, forceDataRecordCreate } from '../../../lib/utils/sfdx/forceDataSoql';
+import { StatusFileManager } from '../../../lib/utils/statusFileManager';
+
+Messages.importMessagesDirectory(__dirname);
+
+const TOPIC = 'extension';
+const CMD = `commerce:${TOPIC}:map`;
+const msgs = Messages.loadMessages('@salesforce/commerce', 'store');
+
+export class MapExtension extends SfdxCommand {
+    public static readonly requiresUsername = true;
+
+    public static description = msgs.getMessage('extension.map.cmdDescription');
+    public static example = [`sfdx ${CMD} --registered-extension-name --store-id --store-name`];
+    public static flagsConfig = {
+        'registered-extension-name': flags.string({
+            char: 'r',
+            description: msgs.getMessage('extension.map.regExtensionNameFlagDescription'),
+        }),
+        'store-name': flags.string({
+            char: 'n',
+            description: msgs.getMessage('extension.map.StoreNameFlagDescription'),
+        }),
+        'store-id': flags.string({
+            char: 'i',
+            description: msgs.getMessage('extension.map.storeId'),
```

Comment:
i am seeing that the format of the flag descriptions start with a lower case letter for commands. 
e.g.
```
./bin/run commerce:store:display --help
Display buyer info

USAGE
  $ sfdx commerce:store:display -n <string> [-b <string>] [-p <string>] [-v <string>] [-u <string>] [--apiversion <string>] [--json] [--loglevel
  trace|debug|info|warn|error|fatal|TRACE|DEBUG|INFO|WARN|ERROR|FATAL]

OPTIONS
  -b, --buyer-username=buyer-username                                               [default: buyer@1commerce.com] buyer's username
  -n, --store-name=store-name                                                       (required) [default: 1commerce] name of the site to create

  -p, --urlpathprefix=urlpathprefix                                                 required if different from store-name URL to append to the domain created when Experiences was
                                                                                    enabled for this org

  -u, --targetusername=targetusername                                               username or alias for the target org; overrides default target org

  -v, --targetdevhubusername=targetdevhubusername                                   username or alias for the dev hub org; overrides default dev hub org

  --apiversion=apiversion                                                           override the api version used for api requests made by this command

  --json                                                                            format output as json

  --loglevel=(trace|debug|info|warn|error|fatal|TRACE|DEBUG|INFO|WARN|ERROR|FATAL)  [default: warn] logging level for this command invocation
```

can we change the descriptions to start with a lowercase letter
```
./bin/run commerce:extension:map --help
Maps an extension to a specific webstore

USAGE
  $ sfdx commerce:extension:map [-r <string>] [-n <string>] [-i <string>] [-u <string>] [--apiversion <string>] [--json] [--loglevel
  trace|debug|info|warn|error|fatal|TRACE|DEBUG|INFO|WARN|ERROR|FATAL]

OPTIONS
  -i, --store-id=store-id                                                           Enter either a name or the store ID
  -n, --store-name=store-name                                                       The name of the webstore to map the extension
  -r, --registered-extension-name=registered-extension-name                         Unique name for extension
  -u, --targetusername=targetusername                                               username or alias for the target org; overrides default target org
  --apiversion=apiversion                                                           override the api version used for api requests made by this command
  --json                                                                            format output as json
  --loglevel=(trace|debug|info|warn|error|fatal|TRACE|DEBUG|INFO|WARN|ERROR|FATAL)  [default: warn] logging level for this command invocation
```